up42/stac.py

```diff
@@ -9,6 +9,12 @@ class InvalidUp42Asset(ValueError):
     pass
 
 
+class IncompleteCollectionDeletionError(ValueError):
```

Comment:
are we sure going for this UX?Because this is a bit contradicting from our Console behavior, where when a user selects and tries to delete an item (which will delete other items in the collection), the UI warns the user but still goes ahead with the deletion, instead of blocking it

Reply:
The SDK is an automation tool, so any confirmation or warning step that we would introduce before deletion would could easily be automated by the user, which could lead to data loss. This was the way we found that would ensure that the user knows exactly what they are deleting.